galette/includes/core_acls.php

```diff
@@ -65,6 +65,7 @@
     'contributions'                     => 'member',
     'printContribution'                 => 'member',
     'myContributions'                   => 'member',
+    'contributionMembers'               => 'groupmanager',
```

Comment:
This route seems not called, either in the plugin (or any other plugin) or in core itself. Did I miss something?

Reply:
In core, in the templates : 
https://github.com/galette/galette/blob/3db69f0f35c6202489c27dc1faabade467f412ba/galette/templates/default/elements/js/choose_adh.js.twig#L27
https://github.com/galette/galette/blob/3db69f0f35c6202489c27dc1faabade467f412ba/galette/templates/default/elements/js/choose_adh.js.twig#L84
https://github.com/galette/galette/blob/3db69f0f35c6202489c27dc1faabade467f412ba/galette/templates/default/elements/js/choose_adh.js.twig#L134

Reply:
Got it, thanks